css/main.scss

```diff
@@ -9,16 +9,38 @@ html {
 }
 
 body {
-	/* we have a top fixed nav bar */
-	margin-top: 50px;
-	margin-bottom: 46px;
+	letter-spacing: .25px;
+    text-rendering: optimizeLegibility;	

```

Comment:
Nit: mixture of tabs and spaces here, better to pick one.

Reply:
Good catch. Fixed.